packages/contracts-bedrock/scripts/DeployConfig.s.sol

```diff
@@ -136,6 +180,62 @@ contract DeployConfig is Script {
         preimageOracleMinProposalSize = stdJson.readUint(_json, "$.preimageOracleMinProposalSize");
         preimageOracleChallengePeriod = stdJson.readUint(_json, "$.preimageOracleChallengePeriod");
         preimageOracleCancunActivationTimestamp = stdJson.readUint(_json, "$.preimageOracleCancunActivationTimestamp");
+
+        //////////////////////////////////////////////////////
+        /// Genesis Config Properties
+        //////////////////////////////////////////////////////
+        l2GenesisBlockNumber = parseJsonUintWithDefault("$.l2GenesisBlockNumber", uint256(0));
+        l2GenesisCanyonTimeOffset = parseJsonUintWithDefault("$.l2GenesisCanyonTimeOffset", uint256(0));
+
+        homesteadBlock = parseJsonUintWithDefault("$.homesteadBlock", uint256(0));
+        eip150Block = parseJsonUintWithDefault("$.eip150Block", uint256(0));
+        eip155Block = parseJsonUintWithDefault("$.eip155Block", uint256(0));
+        eip158Block = parseJsonUintWithDefault("$.eip158Block", uint256(0));
+        byzantiumBlock = parseJsonUintWithDefault("$.byzantiumBlock", uint256(0));
+        constantinopleBlock = parseJsonUintWithDefault("$.constantinopleBlock", uint256(0));
+        petersburgBlock = parseJsonUintWithDefault("$.petersburgBlock", uint256(0));
+        istanbulBlock = parseJsonUintWithDefault("$.istanbulBlock", uint256(0));
+        muirGlacierBlock = parseJsonUintWithDefault("$.muirGlacierBlock", uint256(0));
+        berlinBlock = parseJsonUintWithDefault("$.berlinBlock", uint256(0));
+        londonBlock = parseJsonUintWithDefault("$.londonBlock", uint256(0));
```

Comment:
We should double check which of these we allow configurability for in the existing genesis generation setup and be sure to copy that. I don't think we want to allow older hardforks to not be configured on by default because it creates a footgun for correct configuration, ie there are no checks that enforce older forks are active if a newer fork is active